### ISSUE TITLE
Windows 7 x86 (32bit) build issue

### ISSUE THREAD

**Issue by yurijmikhalevich** [flowplayer/flash-build#13]
Hello.

I'm trying to build FlowPlayer. First time I faced the same issue as in #11. But, when I apply fix with quotes from comments and launch ant.bat I receive next:

```
     [exec] command line: Error: configuration variable 'compiler.define' expected 2 argument(s), got 1
     [exec]
     [exec] Use 'mxmlc -help' for information about using the command line.

BUILD FAILED
C:\Users\d\flash-build\build.xml:108: The following error occurred while executing this line:
C:\Users\d\flash-build\build.xml:173: The following error occurred while executing this line:
C:\Users\d\flash-build\build.xml:230: exec returned: 1
```

Is there any way to fix this? Thank you.

**Comment by yurijmikhalevich** [comment]
It successfully compiles on OS X with JDK 1.6 (only support -d32 argument). Seems, that building is broken on Windows. Also, maybe you should use old JDK version.

**Comment by MudasirBhutto** [comment]
@39dotyt  Were you able to resolve this issue on windows 7? I am also facing same issue. Any hint regarding this?
![issue](https://cloud.githubusercontent.com/assets/5662229/6162722/32c92ba0-b2a9-11e4-8077-369dd80ab614.png)

**Comment by yurijmikhalevich** [comment]
@MudasirBhutto no, as I wrote, I've compiled it on the OS X. From default configuration values it's seems that developers are using OS X. Seems that this issue related to strings escaping, but I'm not sure.

**Comment by MudasirBhutto** [comment]
If same build.xml isn't having any issue regarding string escaping, then why it is occurring on Windows?

**Comment by yurijmikhalevich** [comment]
Maybe, because something in compile environment on Windows treats
strings in other way than on Mac or Linux? 

On 2015-02-13 14:15,
Mudasir Bhutto wrote: 

> If same build.xml isn't having any issue
> regarding string escaping, then why it is occurring on Windows? 
> 
> ## 
> 
> Reply to this email directly or view it on GitHub [1].

## 

web:
https://39.yt/ [2]
jabber: 0@39.yt
skype: thirtyninedotyt

## Links:

[1]
https://github.com/flowplayer/flash-build/issues/13#issuecomment-74232076
[2]
https://39.yt/
